docker_services_cli/env.py

```diff
@@ -133,12 +148,13 @@ def set_env():
 def print_setup_env_config(services, called_from, env_set_command="export"):
     """Prints setup environment instructions."""
     should_print_instructions = False
-    for service in services:
-        for key, value in (
-            SERVICES.get(service)
-            .get("CONTAINER_CONNECTION_ENVIRONMENT_VARIABLES", {})
-            .items()
-        ):
+    for service_type, services_list in services.items():
+        if called_from == "up" and len(services_list) > 1:
+            logging.warning("Multiple %s services %s are being configured. "
```

Comment:
minor: for consistency (as it was used on the rest of the module before) can use `f"...{}"` to format strings.
extra: what is the default logging level? how does the user/tests get or hide this warning.

Reply:
> for consistency (as it was used on the rest of the module before) can use f"...{}" to format strings

`pylint (logging-format-interpolation) (W1202)` and `pylint (logging-not-lazy) W1201`

> extra: what is the default logging level?

Default logging level is `WARNING`

> how does the user/tests get or hide this warning.

I don't think we should allow turning off this logging message, as it can create troubles for developers. The module as it is written can export twice `SQLALCHEMY_DATABASE_URI` and this would only happen if:
- You run `docker-services-cli up --env` without params (boots up everything)
- You run `docker-services-cli up -db mysql --db postgresql --env` (boots up two services of the same type, so you should be warned)
In which cases you really want to see it.